climpred/loadutils.py

```diff
@@ -105,48 +73,57 @@ def open_dataset(name, cache=True, cache_dir=_default_cache_dir,
     Returns:
         The desired xarray dataset.
     """
-    if name.endswith('.nc'):
-        name = name[:-3]
-    # use aliases
-    if name in file_alias_dict.keys():
-        name = file_alias_dict[name]
-    longdir = _os.path.expanduser(cache_dir)
-    fullname = name + '.nc'
-    localfile = _os.sep.join((longdir, fullname))
-    md5name = name + '.md5'
-    md5file = _os.sep.join((longdir, md5name))
-
-    if not _os.path.exists(localfile):
-        # This will always leave this directory on disk.
-        # May want to add an option to remove it.
-        if not _os.path.isdir(longdir):
-            _os.mkdir(longdir)
-
-        if extension is not None:
-            url = '/'.join((github_url, 'raw', branch, extension, fullname))
-            _urlretrieve(url, localfile)
-            url = '/'.join((github_url, 'raw', branch, extension, md5name))
-            _urlretrieve(url, md5file)
-        else:
-            url = '/'.join((github_url, 'raw', branch, fullname))
-            _urlretrieve(url, localfile)
-            url = '/'.join((github_url, 'raw', branch, md5name))
-            _urlretrieve(url, md5file)
-
-        localmd5 = _file_md5_checksum(localfile)
-        with open(md5file, 'r') as f:
-            remotemd5 = f.read()
-        if localmd5 != remotemd5:
+    def get_datasets():
+        """Prints out available datasets for the user to load if no args are
+           given."""
+        for key in file_descriptions.keys():
```

Comment:
I'm wondering if we should use upper case for `file_descriptions` and `file_alias_dict` since both are a constant?
https://www.python.org/dev/peps/pep-0008/#constants

```
FILE_ALIAS_DICT = {'MPI-control-1D': 'PM_MPI-ESM-LR_control',
                   'MPI-control-3D': 'PM_MPI-ESM-LR_control3d',
                   'MPI-PM-DP-1D': 'PM_MPI-ESM-LR_ds',
                   'MPI-PM-DP-3D': 'PM_MPI-ESM-LR_ds3d',
                   'CESM-DP-SST': 'CESM-DP-LE.SST.global',
                   'CESM-DP-SSS': 'CESM-DP-LE.SSS.global',
                   'CESM-LE': 'CESM-LE.global_mean.SST.1955-2015',
                   'ERSST': 'ERSSTv4.global.mean',
                   'FOSI-SST': 'FOSI.SST.global',
                   'FOSI-SSS': 'FOSI.SSS.global'}

file_descriptions = {'MPI-PM-DP-1D': 'decadal prediction ensemble area' +
                                     ' averages of SST/SSS/AMO.',
                     'MPI-PM-DP-3D': 'decadal prediction ensemble' +
                                     ' lat/lon/time of SST/SSS/AMO.',
                     'MPI-control-1D': 'area averages for the control run of' +
                                       ' SST/SSS.',
                     'MPI-control-3D': 'lat/lon/time for the control run of' +
                                       ' SST/SSS.',
                     'CESM-DP-SST': 'decadal prediction ensemble of global' +
                                    ' mean SSTs.',
                     'CESM-DP-SSS': 'decadal prediction ensemble of global' +
                                    ' mean SSS.',
                     'CESM-LE': 'uninitialized ensemble of global mean SSTs.',
                     'ERSST': 'observations of global mean SSTs.',
                     'FOSI-SST': 'reconstruction of global mean SSTs.',
                     'FOSI-SSS': 'reconstruction of global mean SSS.',
                     }
```
Also, I have mixed feelings about nested functions, but it doesn't really matter.